res/layout/game_activity.xml

```diff
@@ -1,21 +1,45 @@
 <?xml version="1.0" encoding="utf-8"?>
-<merge xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
In controller_profile_activity.xml and controller_profile_activity_port.xml:

    IMPORTANT:
    
    Except for the root view, all views in this file MUST set these flags to FALSE:
    
        android:focusable="false"
        android:focusableInTouchMode="false"
    
    This allows focus to stay in the root view at all times.  This is critical because
    the root view is the only view that listens to input from gamepads, keyboards, etc.